### DISCUSSION TITLE
My BlueSCSI sd-card connected to PC + Adaptec controller is seen as a disk!

### DISCUSSION THREAD

**Discussion by ghost** [BlueSCSI/BlueSCSI-v2#105] [Help]
Yea, instead of two virtual disks (two files on the card, called accordingly HD60_512 Amiga and HD10_512 Classic) controller sees the whole 8GB sd-card as a disk and... it's completely usable. Did I something wrong or got some new (another) firmware which allows to use real devices as... real hard-drives?

**Comment by erichelgeson** [comment]
It sounds like it is falling into raw mode using the entire SD card as the device - it does this when no images are found. 

Can you please do a full overwrite format as described in the [wiki troubleshooting page](https://github.com/BlueSCSI/BlueSCSI-v2/wiki/Troubleshooting#what-to-try-first). Place the 2 files back on to the card. Then provide the `log.txt` that was created on the root of the SD card if you're still having an issue.
